### DISCUSSION TITLE
New release

### DISCUSSION THREAD

**Discussion by ChrisTheProgrammer** [Shin-NiL/Godot-Android-Admob-Plugin#183] [General]
Love this plugin. Can we get a 3.3.4 build?

**Comment by Vineuro** [comment]
Would be awesome if we could get a 3.4.2 build.

**Comment by rbarongr** [comment]
I added **TEST ads** successfully to my own project on **Godot 3.4.2**. To test real ads is going to take me much more time but if there's no other news I will share my experience.

I would like to see an 'official' release of this plugin for Godot 3.4.2, but I think everything is working with the current version.

**Reply by Vineuro** [reply]
Would be awesome. Thank you
